frappe/desk/page/setup_wizard/setup_wizard.py

```diff
@@ -417,6 +417,7 @@ def make_records(records, debug=False):
 				raise
 
 		except Exception as e:
+			frappe.db.rollback()
```

Comment:
👍

Reply:
I actually tried this and I guess some silent duplicateentryerrors happen during maria bootstrap because I got a bunch of test failures https://github.com/frappe/erpnext/runs/4661424145?check_suite_focus=true

After I reverted that change the failures went away